mediatypes/image/webp.json

```diff
@@ -11,19 +11,13 @@
             "https://developers.google.com/speed/webp/docs/webp_lossless_bitstream_specification#main-content"
           ],
           "support": {
-            "chrome": [
-              {
-                "version_added": "23"
-              },
-              {
-                "version_added": "17",
-                "version_removed": "23",
-                "partial_implementation": true,
-                "notes": "Lossless compression and alpha transparency are not supported."
-              }
```

Comment:
This note and partial duplicates the `alpha` and `lossless` subfeatures. We don't generally bubble up data like this, so I unwound this.